pkg/graph/resolvers/system_intake_contract_number_test.go

```diff
@@ -0,0 +1,90 @@
+package resolvers
+
+import (
+	"context"
+
+	"github.com/google/uuid"
+	"github.com/jmoiron/sqlx"
+
+	"github.com/cmsgov/easi-app/pkg/appcontext"
+	"github.com/cmsgov/easi-app/pkg/dataloaders"
+	"github.com/cmsgov/easi-app/pkg/models"
+	"github.com/cmsgov/easi-app/pkg/sqlutils"
+	"github.com/cmsgov/easi-app/pkg/testhelpers"
+)
+
+func (s *ResolverSuite) TestContractNumbers() {
+	ctx := context.Background()
+	ctx = appcontext.WithLogger(ctx, s.testConfigs.Logger)
+
+	ctx = dataloaders.CTXWithLoaders(ctx, dataloaders.NewDataLoaders(s.testConfigs.Store, func(ctx context.Context, s []string) ([]*models.UserInfo, error) { return nil, nil }))
+
+	const (
+		contract1 = "1"
+		contract2 = "2"
+		contract3 = "3"
+	)
+
+	var createdID uuid.UUID
+
+	// create system intake
+	s.Run("create system intake for test", func() {
```

Comment:
We happen to run our tests in sequence (not in parallel), so this is practically safe, but executions of `s.Run()` are often best suited to individual test cases.

You could either pull all of this into a single `s.Run()` (that sets up the data, sets contract numbers, and validates them), or just get rid of the `s.Run()` calls all-together

Reply:
Done! Thanks!